### ISSUE TITLE
Feature: Advanced Data Import

### ISSUE THREAD

**Issue by frederikbugglin** [basxsoftwareassociation/basxconnect#124]
https://flatfile.com/product/portal/#features

**Comment by saemideluxe** [comment]
This is interesting but I am a bit concerned about flatfile sending import-data to their own servers.

**Comment by frederikbugglin** [comment]
> This is interesting but I am a bit concerned about flatfile sending import-data to their own servers.

@saemideluxe I fully agree on that. However, I will leave this issue open as we previously have discussed an advanced data import feature. OK?

**Comment by saemideluxe** [comment]
Yes, sure. I definitely would love to see a better solution for data import than some custom scripts.
